inventory-generation/identity-management/main.yml

```diff
@@ -9,6 +9,10 @@
       when:
         - directory is undefined or (directory | trim) == ""
 
+    - name: Read Engagement Data
```

Comment:
This move seems unnecessary - unless there's a reason I'm missing, please undo this change

Reply:
OK, great - thx for the explanation